src/main/java/com/wavefront/sdk/common/clients/WavefrontClient.java

```diff
@@ -264,6 +271,16 @@ private WavefrontClient(Builder builder) {
         sendSdkMetrics(builder.includeSdkMetrics).
         build();
 
+    try {
+      final Properties properties = new Properties();
+      properties.load(WavefrontClient.class.getResourceAsStream("/build.properties"));
+      String version = properties.getProperty("version");
+      double sdkVersion = getSemVer(version);
```

Comment:
Is it possible to factor this logic out so that it isn't repeated in each client?  Can perhaps use `ResourceBundle.getBundle("build")` to get build properties as opposed to using the class.

Reply:
> Should we be consistent with proxy, which would emit `2006001` for `2.6.1`?

I think the proxy way of reporting build versions or our cluster might not be suitable to represent semantic versions.
I felt this is a better way to represent semvers without a lot of calculation logic , and more friendly to understand when looking at the chart instead of 2006001 (Also the proxy way, will need redundant calculation on the charts to decipher what the patch version is say distinguish between 1 and 10. Note we cant do modulo expression today in charts.)

"<major>.<2-digit-minor><2-digit-patch>"

Reply:
sounds good